packages/extension/src/components/pages/AddNewTrustline/StepForm.tsx

```diff
@@ -0,0 +1,133 @@
+import { FC, FocusEvent, useCallback, useMemo, useState } from 'react';
+
+import { Button, TextField } from '@mui/material';
+import { useNavigate } from 'react-router-dom';
+
+import { HOME_PATH } from '../../../constants';
+import { NumericInput } from '../../atoms';
+import { PageWithReturn } from '../../templates';
+
+const MAX_TOKEN_LENGTH = 30;
```

Comment:
In packages/extension/src/components/pages/AddNewTrustline/AddNewTrustline.tsx
line 237, you have set a max length of 40. Maybe it should be 40 here as well.

Maybe this constant can be extracted and used in both this file and packages/extension/src/components/pages/AddNewTrustline/AddNewTrustline.tsx

You can put this constant in: packages/extension/src/constants

Reply:
the 40 you saw is not a limit but actually to detect if we have received an already HEX encoded currency or not, since the XRPL requires the token which size > 3 to be HEX encoded into a hex with a size of exactly 40.
So I check if what I received has the size 40 or not in order to not double hex encode these.

But overall I agree to move this const

Reply:
Done